### ISSUE TITLE
Xcode 12.0编译报错

### ISSUE THREAD

**Issue by allocolloid** [jpush/aurora-imui#687]
### My environment:

<img width="1145" alt="截屏2020-09-18 15 38 53" src="https://user-images.githubusercontent.com/12625534/93570642-e4293200-f9c5-11ea-96e0-e79043069309.png">

**Comment by zerophilip** [comment]
Same problem, help!!

**Comment by zerophilip** [comment]
Change it   

```
//      let photoSettings = AVCapturePhotoSettings()
//      photoSettings.flashMode = .auto
//      photoSettings.isHighResolutionPhotoEnabled = false
//
//      if photoSettings.availablePreviewPhotoPixelFormatTypes.count > 0 {
//        photoSettings.previewPhotoFormat = [kCVPixelBufferPixelFormatTypeKey as String : photoSettings.availablePreviewPhotoPixelFormatTypes.first!]
//      }
      
var photoSettings: AVCapturePhotoSettings
       
                if #available(iOS 11.0, *) {
                    photoSettings = AVCapturePhotoSettings.init(format: [AVVideoCodecKey: AVVideoCodecType.jpeg])
                } else {
                    // Fallback on earlier versions
                    photoSettings = AVCapturePhotoSettings()
                    if photoSettings.__availablePreviewPhotoPixelFormatTypes.count > 0 {
                        photoSettings.previewPhotoFormat = [kCVPixelBufferPixelFormatTypeKey as String : photoSettings.__availablePreviewPhotoPixelFormatTypes.first!]
                    }
                }
```

**Comment by MShuai168** [comment]
> Change it
> 
> ```
> //      let photoSettings = AVCapturePhotoSettings()
> //      photoSettings.flashMode = .auto
> //      photoSettings.isHighResolutionPhotoEnabled = false
> //
> //      if photoSettings.availablePreviewPhotoPixelFormatTypes.count > 0 {
> //        photoSettings.previewPhotoFormat = [kCVPixelBufferPixelFormatTypeKey as String : photoSettings.availablePreviewPhotoPixelFormatTypes.first!]
> //      }
>       
> var photoSettings: AVCapturePhotoSettings
>        
>                 if #available(iOS 11.0, *) {
>                     photoSettings = AVCapturePhotoSettings.init(format: [AVVideoCodecKey: AVVideoCodecType.jpeg])
>                 } else {
>                     // Fallback on earlier versions
>                     photoSettings = AVCapturePhotoSettings()
>                     if photoSettings.__availablePreviewPhotoPixelFormatTypes.count > 0 {
>                         photoSettings.previewPhotoFormat = [kCVPixelBufferPixelFormatTypeKey as String : photoSettings.__availablePreviewPhotoPixelFormatTypes.first!]
>                     }
>                 }
> ```

Due to the new features iOS14, when selecting a photo album , the following code should modify。If they don't update, I will change the other similar to the SDK。

![1](https://user-images.githubusercontent.com/12625534/93855121-9116ee00-fce9-11ea-8eda-1d3612908145.jpg)
![2](https://user-images.githubusercontent.com/12625534/93855133-95430b80-fce9-11ea-8689-15b7ac85a663.jpg)
